packages/app/src/views/Home.vue

```diff
@@ -1,34 +1,49 @@
 <template>
-  <div class="home mx-auto max-w-lg h-full w-full flex flex-col">
-    <header class="h-1/5 flex items-end justify-center text-4xl text-gray-800">
+  <div class="home flex flex-col">
```

Comment:
```suggestion
  <div class="home w-full flex flex-col">
```
I think there is just a w-full missing

Reply:
Stange I do not have this error on any of my responsive stages, but adding `w-full` does not break it.